### PR TITLE
Add Digital Federal Credit Union into Banking

### PR DESCRIPTION
**Pull request by vgarmash**

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -202,6 +202,17 @@ websites:
       facebook: Deutsche.Kreditbank
       lang: de
 
+    - name: Digital Federal Credit Union
+      url: https://www.dcu.org
+      img: dcu.png
+      tfa: yes
```
Comment (Jawshy) [inline _data/banking.yml]: Please change to `Yes`.

**Comment by Jawshy** [comment]
@vgarmash, the `twitter` and `facebook` tags are not needed when the `tfa` tag is set as `Yes`. Remove them and the build should pass.

File: _data/banking.yml (hunk 1 above)
Comment (vgarmash) [reply _data/banking.yml]: Done

**Comment by vgarmash** [comment]
I squashed all changes into 1 single commit.

**Comment by Jawshy** [comment]
Excellent - build passed. :+1:

**Review by stephengroat — APPROVED** [review]

**Pull request merged**